drivers/storage/portworx/portworx_test.go

```diff
@@ -163,6 +163,45 @@ func TestValidate(t *testing.T) {
 	require.Contains(t, <-recorder.Events,
 		fmt.Sprintf("%v %v %s", v1.EventTypeNormal, util.PassPreFlight, "Enabling PX-StoreV2"))
 	require.Contains(t, *cluster.Spec.CloudStorage.SystemMdDeviceSpec, DefCmetaData)
+
+	//
+	// Validate Pre-flight Daemonset Pod Spec
+	//
+	cluster.Spec.Image = "portworx/oci-image:3.0.0"
+	cluster.Annotations = map[string]string{
+		pxutil.AnnotationPreflightCheck: "true",
+	}
+	err = driver.SetDefaultsOnStorageCluster(cluster)
+	require.NoError(t, err)
+
+	podSpec, err := driver.GetStoragePodSpec(cluster, "")
+	require.NoError(t, err)
+
+	// phone-home cm volume mount, should not exists since pre-flight is enabled
+	var errChk error
+	for _, volumeMount := range podSpec.Containers[0].VolumeMounts {
+		if volumeMount.Name == "ccm-phonehome-config" {
```

Comment:
nit: you could replace the block with the assert-check (also, check all the containers) -- e.g. :

```go
	// phone-home cm volume mount, should not exists since pre-flight is enabled
	for _, cnt := range podSpec.Containers {
	    for _, volumeMount := range cnt.VolumeMounts {
		require.True(t, volumeMount.Name != "ccm-phonehome-config")
            }
        }
        // also potentially
        require.Equals(t, 1, len(podSpec.Containers))
```

Reply:
I can make that change .. thanks @zoxpx